src/core/link.h

```diff
@@ -9,7 +9,10 @@ struct link {
   //   http://en.wikipedia.org/wiki/Circular_buffer
   struct packet *packets[LINK_RING_SIZE];
   struct {
-    struct counter *dtime, *txbytes, *rxbytes, *txpackets, *rxpackets, *txdrop;
+    struct counter *dtime,
+                   *input_bytes, *output_bytes,
+                   *input_packets, *output_packets,
+                   *input_drop;
```

Comment:
Is the naming of `input` vs `output` clear to everybody else?

Previously we had `link.transmit()` incrementing the `txpackets` which seems like a direct match. Now we have `link..transmit()` incrementing the `input_packets` which seems more ambiguous to me. (Does the sender input packets _onto_ a link, or does the receiver input packets _from_ a link?)

If this is just me then it doesn't matter but otherwise we could consider e.g. `enqueue` and `dequeue` as perhaps being harder to misinterpret.